packages/origin-ui-core/src/components/Table/TableMaterial.tsx

```diff
@@ -64,7 +64,7 @@ interface IProps<T extends readonly ITableColumn[]> {
     loadPage?: (page: number, filters?: ICustomFilter[]) => void | Promise<any>;
     pageSize?: number;
     total?: number;
-    actions?: ITableAction[];
+    actions?: ITableAction[] | ITableAction[][];
```

Comment:
I'm trying to understand why we have a 2D array of `ITableAction`. What's the reason for it?